README.md

```diff
@@ -94,10 +100,28 @@ In order to run this project you need:
 Clone this repository to your desired folder using:
 `git clone https://github.com/supreeti/OOPS-Library-Management-System.git` on your terminal.
 
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
```

Comment:
- [ ] Since the `main.rb` file is your app entry point, I think the right command for running your project is `ruby main.rb` instead or `rails server` :blush: